e2e/single-cluster/gitrepo_polling_disabled_test.go

```diff
@@ -67,7 +68,17 @@ var _ = Describe("GitRepoPollingDisabled", Label("infra-setup"), func() {
 			"fleet-controller",
 		)
 		Expect(err).ToNot(HaveOccurred())
-		Expect(out).ToNot(ContainSubstring("ERROR"))
+
+		// Errors about bundles or bundle deployments not being found at deletion time should be ignored.
+		isError, err := regexp.MatchString(
```

Comment:
I had trouble understanding this until I looked at the commit message. I still think the code comment here doesn't match the commit message?

> Checking logs for a single `ERROR` keyword may lead to false positives,
for instance caused by other test suites. Instead, this uses a regular
expression to look for logs related to missing bundles or bundle
deployments.


Maybe getting rid of one more error check with `regexp.MustCompile` or:

```
Expect(out).ToNot(MatchRegexp(`ERROR.*Reconciler error.*Bundle(Deployment)?.fleet.cattle.io \\".*\\" not found`))
```

Reply:
Thanks, comment fixed and regex check simplified.